kube-client/src/config/file_config.rs

```diff
@@ -249,6 +249,23 @@ pub struct ExecConfig {
     /// It has been suggested in client-go via https://github.com/kubernetes/client-go/issues/1177
     #[serde(skip)]
     pub drop_env: Option<Vec<String>>,
+
+    /// Interative mode of the auth plugins
+    #[serde(rename = "interactiveMode")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub interactive_mode: Option<ExecInteractiveMode>,
```

Comment:
looks good, matches upstream!

there are a couple of extra properties related to this https://github.com/kubernetes/client-go/blob/master/tools/clientcmd/api/types.go#L257-L282 do you think they are wroth doing here while we are touching this?

Reply:
resolving, this stands on its own